### PR TITLE
Option to disable  TaskUnobservedTaskException Capture

### PR DESCRIPTION
**Pull request by FilipNemec**
Option to disable  TaskUnobservedTaskException Capture thorigh SentryOptions method DisableTaskUnobservedTaskExceptionCapture. This option will enable that sentry doesn't receive TaskUnobservedTaskException automatically.

### CODE REVIEW

**Comment by FilipNemec** [comment]
Regarding the issue https://github.com/getsentry/sentry-dotnet/issues/849

**Review by bruno-garcia — COMMENTED** [review]
It all looks great but it's missing adding it in the constructor of SentryOptions. The method you added will allow users to opt-out of that.
Also, please add an entry to the CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Adding TaskUnobservedTaskExceptionIntegration to default integrations and method to remove it @FilipNemec
```
Comment (bruno-garcia) [inline CHANGELOG.md:7]:
```suggestion
- Adding TaskUnobservedTaskExceptionIntegration to default integrations and method to remove it (#870) @FilipNemec
```

**Review by bruno-garcia — APPROVED** [review]
Awesome! Thanks for contributing!

**Pull request merged**